yasa/plotting.py

```diff
@@ -6,42 +6,29 @@
 import pandas as pd
 import seaborn as sns
 import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
 from lspopt import spectrogram_lspopt
 from matplotlib.colors import Normalize, ListedColormap
 
 __all__ = ["plot_hypnogram", "plot_spectrogram", "topoplot"]
 
 
-def plot_hypnogram(hypno, sf_hypno=1 / 30, lw=1.5, fill_color=None, ax=None):
+def plot_hypnogram(hyp, lw=1, fill_color=None, highlight=None, ax=None):
     """
     Plot a hypnogram.
 
     .. versionadded:: 0.6.0
 
     Parameters
     ----------
-    hypno : array_like
-        Sleep stage (hypnogram).
-
-        .. note::
-            The default hypnogram format in YASA is a 1D integer vector where:
-
-            * -2 = Unscored
-            * -1 = Artefact / Movement
-            * 0 = Wake
-            * 1 = N1 sleep
-            * 2 = N2 sleep
-            * 3 = N3 sleep
-            * 4 = REM sleep
-    sf_hypno : float
-        The current sampling frequency of the hypnogram, in Hz, e.g.
-
-        * 1/30 = 1 value per each 30 seconds of EEG data,
-        * 1 = 1 value per second of EEG data
+    hyp : :py:class:`yasa.Hypnogram`
+        A YASA hypnogram instance.
     lw : float
         Linewidth.
     fill_color : str
-        Color to fill space above hypnogram line, optional.
+        Optional color to fill space above hypnogram line.
+    highlight : str
+        Optional stage to highlight with alternate color.
```

Comment:
Yes we can do that. At first I was concerned about rare cases where `n_stages` doesn't include REM, but I think they way it's set up now it will just highlight nothing (as opposed to erroring out). Note this will also auto-highlight REM when `n_stages` is 3 or 4. I don't know if that's a norm or not.

Reply:
> Note this will also auto-highlight REM when n_stages is 3 or 4

I think that's fine! If that's okay with you then I vote for REM as default